components/forms/Form.js

```diff
@@ -1,43 +1,47 @@
 import styled from "styled-components";
 
-export default function WeightForm({ onAddWeight }) {
+export default function Form({ onAddData }) {
   function sendForm(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
-    const { weight, date } = Object.fromEntries(formData);
+    const { weight, date, height } = Object.fromEntries(formData);
 
-    onAddWeight(weight, date);
+    onAddData(weight, date, height);
   }
   return (
-    <WeightFormElement aria-label="Add weight and date" onSubmit={sendForm}>
+    <FormElement aria-label="Add weight and date" onSubmit={sendForm}>
+      <input
+        aria-label="date input"
+        type="date"
+        name="date"
+        defaultValue={new Date().toISOString().slice(0, 10)}
+      />
       <input
         type="number"
         name="weight"
         min="0"
+        max="50"
         step="0.001"
         aria-label="weight input"
```

Comment:
Maybe you can consider to add a function to switch between 0.001 and maybe 0.01 and 1 steps depending on baby age or something like that for later user story's

Reply:
sounds like a nice feature for the near future, thanks!